src/index.html

```diff
@@ -79,7 +80,90 @@ <h2 class="title title--secondary">Ayurvedic Tooth Powder</h2>
         </div>
         <p class="title__highlight">learn more</p>
       </section>
+
+      <section class="signs signs--section">
+        <img src="./images/Group 10.svg" alt="10"
+        class="signs__picture">
+        <p class="signs__paragraph">No synthetics and microplastics</p>
+        <img src="./images/Group 9.svg" alt="10"
+        class="signs__picture">
+        <p class="signs__paragraph">No synthetics and microplastics</p>
+        <img src="./images/Group 11.svg" alt="10"
+        class="signs__picture">
+        <p class="signs__paragraph">No synthetics and microplastics</p>
+      </section>
+
+      <section class="contact-us">
+        <h2 class="title title--secondary">Contact us</h2>
+        <p class="contact-us__paragraph">
+          We will help you choose a product that suits you and answer your other questions
+        </p>
+        <form class="form">
+
+          <div class="form__item">
+            <label for="name" class="form__label">Your Name</label>
+            <input type="text"
+            placeholder="Your Name"
+            class="form__input"
+            id="name"
+            required>
+          </div>
+
+          <div class="form__item">
+            <label for="email" class="form__label">E-Mail</label>
+            <input type="email"
+            placeholder="E-Mail"
+            class="form__input"
+            id="email"
+            required>
+          </div>
+
+          <div class="form__item">
+            <label for="number" class="form__label">Phone Number</label>
+            <input type="number"
+            placeholder="Phone Number"
+            class="form__input"
+            id="number"
+            required>
+          </div>
+
+          <div class="form__item">
+            <label for="text" class="form__label">Your Message</label>
+            <textarea class="form__text" id="text"
+            placeholder="Your Message"></textarea>
+          </div>
+
+
+          <a href="#" class="button button--primary">send a message</a>
+        </form>
+      </section>
     </main>
+    <footer class="footer">
+      <section class="section__footer">
+        <ul class="list-sec">
+          <li class="list-sec__list"><a href="#shop" class="list-sec__link">shop</a></li>
+          <li class="list-sec__list"></li><a href="#contact-us"class="list-sec__link">contact</a></li>
+          <li class="list-sec__list"></li><a href="#about"class="list-sec__link">About</a></li>
```

Comment:
you close li tag two times
```suggestion
          <li class="list-sec__list"><a href="#contact-us"class="list-sec__link">contact</a></li>
          <li class="list-sec__list"><a href="#about"class="list-sec__link">About</a></li>
```